spring-cloud-kubernetes-config/src/main/java/org/springframework/cloud/kubernetes/config/ConfigUtils.java

```diff
@@ -39,38 +39,33 @@ private ConfigUtils() {
 		throw new IllegalStateException("Can't instantiate a utility class");
 	}
 
-	public static <C extends AbstractConfigProperties> String getApplicationName(
-			Environment env, String configName, String configurationTarget) {
-		String name = configName;
-		if (StringUtils.isEmpty(name)) {
+	public static String getApplicationName(Environment env, String configName,
```

Comment:
- un-used type parameter
 - Strings are immutable, no need for the local copy
 - the check if `Log::debug` is enabled is already done _inside_ `Log::debug`; no need to do that yourself. The only time this does make sense, is when the message itself is expensive to compute, but that is by far not the case here